lib/tmp.js

```diff
@@ -610,43 +617,33 @@ function _is_legacy_listener(listener) {
  */
 function _safely_install_sigint_listener() {
 
-  let listeners = process.listeners(SIGINT);
+  const listeners = process.listeners(SIGINT);
   const existingListeners = [];
   for (let i = 0, length = listeners.length; i < length; i++) {
     const lstnr = listeners[i];
     /* istanbul ignore else */
-    if (lstnr.name === SIGINT_LISTENER_NAME) {
+    if (lstnr.name === '_tmp$sigint_listener') {
       existingListeners.push(lstnr);
       process.removeListener(SIGINT, lstnr);
     }
   }
-  process.on(SIGINT, function _tmp$sigint_listener() {
+  process.on(SIGINT, function _tmp$sigint_listener(doExit) {
```

Comment:
This variables `doExit` will just be the string `SIGINT`. Meaning this will _always_ exit on line 644.

Reply:
This has been resolved as we do non longer add a SIGINT handler.

Regardless, the logic behind this was to postpone exiting to the very last sigint handler, supporting multiple legacy versions of tmp here.